.gitignore

```diff
@@ -43,7 +192,7 @@ htmlcov/
 .cache
 nosetests.xml
 coverage.xml
-*.cover
+*,cover
```

Comment:
yes but the generated gitignore does have a comma there. And that's not an extension actually, just a filename. I'm not sure what it is used for.